### ISSUE TITLE
Sharing hand and object bounding boxes for EGTEA 

### ISSUE THREAD

**Issue by debadityaroy** [stevenlsw/hoi-forecast#2]
Hi Steven

Thanks for sharing this interesting work. In the paper, you have extracted hand and object bounding boxes for the datasets. 

Is it possible to share it for EGTEA dataset as it is not publicly available?

Thanks and Regards
Debaditya

**Comment by stevenlsw** [comment]
So sorry for the late reply, but the dataset is available online at https://cbs.ic.gatech.edu/fpv/

**Comment by debadityaroy** [comment]
Thanks for providing the detection results.

**Comment by debadityaroy** [comment]
Hi Steven
How do we connect the video_id in the pkl file to the original videos?

**Comment by stevenlsw** [comment]
Hi, you could find the corresponding labels here https://drive.google.com/drive/folders/1lNOSXXKbiqYJqC1hp1CIaIeM8u6UVvcg, the full information is available at 
the project page https://stevenlsw.github.io/hoi-forecast/ under training data.

**Comment by debadityaroy** [comment]
Thanks Steven

**Comment by debadityaroy** [comment]
Hi Steven

Thanks for pointing to the labels. 
However, the uids are only available for 2322 files in egtea.csv. There are a total of 10321 pickle files. Do you have the rest of the mappings somewhere?

**Comment by stevenlsw** [comment]
The uids is the same order as the following start from 0
